src/Sets/SparsePolynomialZonotope.jl

```diff
@@ -397,3 +397,35 @@ function reduce_order(P::SparsePolynomialZonotope, r::Real, method::AbstractRedu
     Gz = genmat(Z)
     return SparsePolynomialZonotope(cz, G[:, Kbar], hcat(GI[:, Hbar], Gz), Ebar[N, :], idx[N])
 end
+
+"""
+    ρ(d::AbstractVector, P::SparsePolynomialZonotope{N}; method)
+
+Bound the support function over ``P`` in the direction ``d``.
+"""
+@inline function ρ(d::AbstractVector, P::SparsePolynomialZonotope{N}; method=nothing) where {N}
+    require(@__MODULE__, :RangeEnclosures; fun_name="ρ")
+    return _ρ_range_enclosures(d, P, method)
+end
+
+function _load_rho_range_enclosures()
+    return quote
+        function _ρ_range_enclosures(d::AbstractVector, P::SparsePolynomialZonotope, method::Union{RangeEnclosures.AbstractEnclosureAlgorithm, Nothing})
```

Comment:
We usually do not indent these `Requires` blocks. So it could just be

```julia
function _load_rho_range_enclosures()
return quote

function _ρ_range_enclosures(...
    ...
end

end  # quote
end  # function _load_rho_range_enclosures()

```